instaclustr/resource_vpc_peering.go

```diff
@@ -139,9 +144,7 @@ func resourceVpcPeeringRead(d *schema.ResourceData, meta interface{}) error {
 	d.Set("peer_vpc_id", vpcPeering.PeerVpcID)
 	d.Set("peer_account_id", vpcPeering.PeerAccountID)
 	d.Set("aws_vpc_connection_id", vpcPeering.AWSVpcConnectionID)
-
-	before, _ := d.GetChange("peer_subnet")
-	d.Set("peer_subnet", before)
+	d.Set("peer_subnet", vpcPeering.PeerSubnet)
```

Comment:
Curious about this change, i haven't look to deeply yet. But what was the reason behind it?

Reply:
Good question, when importing resources, the ResourceData would be initialised with only ID set, so before in "before, _ := d.GetChange("peer_subnet")" will always be nil when importing;

Set would be more applicable for resourceRead.

Reply:
Sounds reasonable to me 👍🏼